timemachine/fe/interpolate.py

```diff
@@ -150,7 +150,7 @@ def linear_interpolation(src_params, dst_params, lamb):
     """
     Linearly interpolate between src and dst params
     """
-    return (1 - lamb) * src_params + lamb * dst_params
+    return jnp.where(src_params == dst_params, src_params, (1 - lamb) * src_params + lamb * dst_params)
```

Comment:
Might add a comment on why this is necessary?

Reply:
added in https://github.com/proteneer/timemachine/pull/1538/commits/f357272df94849dfbb05722c41c23af0452ec81d